.gitlab.mk

```diff
@@ -117,8 +117,11 @@ vms_shutdown:
 GIT_DESCRIBE=$(shell git describe HEAD)
 GIT_TAG=$(shell git tag --points-at HEAD)
 MAJOR_VERSION=$(word 1,$(subst ., ,$(GIT_DESCRIBE)))
-BUCKET="series-$(MAJOR_VERSION)"
+TNT_VERSION_S3_DIR=series-$(MAJOR_VERSION)
```

Comment:
Will we backport this branch for avoiding the support of update_repo.sh? 
In this case, please, be accurate with `TNT_VERSION_S3_DIR`.

Reply:
Yes, I think we should backport it.

Reply:
Raw idea: implement universal logic here that will work good for all branches and don't be afraid about backports.

Reply:
Done